src/test/java/org/commcare/util/GetJsonPropertyTest.java

```diff
@@ -0,0 +1,18 @@
+package org.commcare.util;
+
+import org.junit.Test;
+import org.junit.Assert;
+
+import static org.junit.Assert.assertTrue;
+
+public class GetJsonPropertyTest {
+
+    @Test
+    public void getJsonProperty() {
+        String testObj1 = "{name: Sam}";
+        String testVal1 = GetJsonProperty.getJsonProperty(testObj1, "name");
```

Comment:
A couple of negative tests would be nice documentation here: invalid json, a more deeply nested object.